src/app/common/stores/Overview.store.ts

```diff
@@ -22,17 +23,24 @@ class OverviewStore extends BaseStore {
 
   @action.bound
   setTotalEth(totalEth: number) {
+    let unversionUrl = config.links.API_COMPLETE_BASE_URL;
+
+    const pathnames = unversionUrl.split('/');
+    pathnames.splice(-2);
```

Comment:
There is more stable and deterministic way of how to do this:

```javascript
let str = 'http://api.stage.ssv.network/api/v4/prater/something%20else?foo=bar';
let newStr = str.replace(/(\/api\/).*/, '$1').replace(/\/+$/, '');
console.log(newStr);
```

Reply:
done